scripts/gen_accounts.js

```diff
@@ -0,0 +1,52 @@
+const { Personal } = require('web3-eth-personal');
+const { Web3Eth } = require('web3-eth');
+const tempAccountList = require('./accounts.json');
+
+const getEnvVar = name => (global.Cypress ? Cypress.env(name) : process.env[name]);
+
+const DEFAULT_SYSTEM_PROVIDER = 'http://localhost:8545';
+
+const getSystemTestProvider = () => DEFAULT_SYSTEM_PROVIDER;
+
+const getSystemTestBackend = () => getEnvVar('WEB3_SYSTEM_TEST_BACKEND') ?? '';
+
+let mainAcc;
+let accountList = [];
+const addAccount = async (address, privateKey) => {
+	let clientUrl = getSystemTestProvider();
+
+	const web3Personal = new Personal(clientUrl);
+	if (accountList.length === 0) {
+		accountList = await web3Personal.getAccounts();
+		mainAcc = accountList[0];
+	}
+	const web3Eth = new Web3Eth(clientUrl);
+
+	if (!accountList.find(acc => acc.address === address)) {
+		await web3Personal.importRawKey(
```

Comment:
For future test cases we will need to use and test local wallet along with most of tests, instead of node wallet. So some future tests scenarios will use local signing functionality instead of loading PK into node wallet, unlocking and sending unsigned transactions. 
This PR is waiting from long time so we can have above mentioned functionality in some next sprint.

Reply:
yes. I think, we have a separate task for this feature